src/jit/lowerxarch.cpp

```diff
@@ -3603,6 +3614,41 @@ void Lowering::TreeNodeInfoInitCmp(GenTreePtr tree)
                 if (!op1IsMadeContained)
                 {
                     SetRegOptional(op1);
+
+                    // If op1 codegen sets ZF and SF flags and ==/!= against
```

Comment:
Very minor suggestion: Since the code generally extends to about column 120 or so, it would be good to use more of the horizontal space for the comment, so that one can see more of the comment and code in a single screen.